content/_meta/info/legal.njk

```diff
@@ -8,7 +8,7 @@ est: |
 
 {{ est | md | safe }}
 
-<p class="h-card">
+<p class="h-card p-org">
```

Comment:
I'm not sure this is doing anything, since it's already a `p` (and won't be bold)?

Reply:
That's part of the h-card microformat for organizations. I didn't add it for the sake of style, but for the sake of completing the pattern.